py3status/modules/netdata.py

```diff
@@ -59,12 +59,22 @@ class Py3status:
     low_traffic = 400
     med_speed = 60
     med_traffic = 700
-    nic = 'eth0'
+    nic = None
 
     def __init__(self):
         self.old_transmitted = 0
         self.old_received = 0
 
+    def post_config_hook(self):
+        if self.nic is None:
+            """ Get default gateway directly from /proc."""
+            with open("/proc/net/route") as fh:
+                for line in fh:
+                    fields = line.strip().split()
+                    if fields[1] != '00000000' or not int(fields[3], 16) & 2:
+                        continue
+                    self.nic = fields[0]
```

Comment:
```
if fields[1] == '00000000' or int(fields[3], 16) & 2:
        self.nic = fields[0]
        break
```
I undo the above. I tried with what you wanted me to do here and couldn't get it to work. I'm guessing something like the code above. What do I do to optimize this better?